bc_obps/registration/models/closure_event.py

```diff
@@ -0,0 +1,34 @@
+from registration.models import Event, Operation, Facility
+from django.db import models
+from django.core.exceptions import ValidationError
+from simple_history.models import HistoricalRecords
+
+
+class Closure(Event):
```

Comment:
I suggest keeping the class name the same as the file name, like `ClosureEvent`. This way, we can use `related_name="closure_events"`, which I believe makes more sense.

Reply:
The same applies to other data models as well.